net-analyzer/nrpe/nrpe-4.0.2-r1.ebuild

```diff
@@ -0,0 +1,105 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit autotools systemd
+
+DESCRIPTION="Nagios Remote Plugin Executor"
+HOMEPAGE="https://github.com/NagiosEnterprises/nrpe"
+SRC_URI="https://github.com/NagiosEnterprises/nrpe/releases/download/${P}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~ppc ~ppc64 ~sparc ~x86"
+IUSE="command-args libressl selinux ssl tcpd"
+
+DEPEND="acct-group/nagios
+	acct-user/nagios
+	tcpd? (
+		sys-apps/tcp-wrappers
+	)
+	ssl? (
+		!libressl? ( dev-libs/openssl:0= )
+		libressl? ( dev-libs/libressl:0= )
+	)"
+RDEPEND="${DEPEND}
+	|| ( net-analyzer/nagios-plugins net-analyzer/monitoring-plugins )
+	selinux? ( sec-policy/selinux-nagios )"
```

Comment:
I'm not really happy seeing this USE flag having no other function in this ebuild. Could you move it to `optfeature`?

Reply:
Did a quick grep for selinux on the tree, not seeing optfeature used for this at all, all selinux USE flag uses the structure here.  But I think I get why you're saying this ... if the USE flag change, we get a pointless recompile.  Fortunately selinux is one of those that should ALMOST NEVER change.

Reply:
You are correct, I was kind of 50-50 about that comment since selinux is a bit special, and straight out controlled by profiles. You can safely presume it's globally either enabled or disabled.